### ISSUE TITLE
Website Contains Broken Links

### ISSUE THREAD

**Issue by hexsecs** [uptane/uptane.github.io#123]
## Website Contains Broken Links

Broken Link Checker found :coffin: links on https://uptane.github.io

[View Results](https://github.com/uptane/uptane.github.io/actions/runs/6714594355?check_suite_focus=true)

Here is list of broken links : 

- [ ] Link 1 - code:    404
  url:     https://www.d2pmagazine.com/2020/04/02/6099/
  parent:  https://uptane.github.io/docs/learn-more/press
  error:   Not Found

- [ ] Link 2 -  code:    404
  url:     https://WZ6MBDOHOT-dsn.algolia.net
  parent:  https://uptane.github.io
  error:   Not Found

- [ ] Link 3 -  code:    404
  url:     https://uptane.github.io/participate.html
  parent:  https://uptane.github.io/docs/learn-more/audits
  error:   Not Found

- [ ] Link 4 - code:    404
  url:     https://uptane.github.io/pouf.html
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- [ ] Link 5 - code:    404
  url:     https://uptane.github.io/reference_pouf.html
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- [ ] Link 6 - [ ]  code:    404
  url:     https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- [ ] Link 7 - code:    404
  url:     https://uptane.github.io/deployment/introduction.html
  parent:  https://uptane.github.io/docs/1.0.0/standard/uptane-standard
  error:   Not Found

- [ ] Link 8 - code:    404
  url:     https://uptane.github.io/attacks.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 9 - code:    404
  url:     https://uptane.github.io/overview.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 10 - code:    404
  url:     https://uptane.github.io/design.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 11 - code:    404
  url:     https://uptane.github.io/governance.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 12 - code:    404
  url:     https://uptane.github.io/timeline.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 13 - code:    404
  url:     https://uptane.github.io/adoptions.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 14 - code:    404
  url:     https://uptane.github.io/people.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 15 - code:    404
  url:     https://uptane.github.io/standardref.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 16 - code:    404
  url:     https://uptane.github.io/publications.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 17 - code:    404
  url:     https://uptane.github.io/audio.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 18 - code:    404
  url:     https://uptane.github.io/audits.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 19 - code:    404
  url:     https://uptane.github.io/alliance.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 20 - code:    404
  url:     https://uptane.github.io/press.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- [ ] Link 21 - code:    403
  url:     https://rin.org.uk/blogpost/[17](https://github.com/uptane/uptane.github.io/actions/runs/6714594355/job/18248060571#step:5:18)06945/332376/What-is-spoofing-and-how-to-ensure-GPS-security
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Forbidden

- [ ] Link 22 - code:    403
  url:     https://knowhow.napaonline.com/how-many-parts-are-in-a-car/
  parent:  https://uptane.github.io//docs/2.0.0/deployment/best-practices#10-changelog
  error:   Forbidden

- [ ] Link 23 - code:    400
  url:     https://www.googletagmanager.com
  parent:  https://uptane.github.io
  error:   Bad Request

- [ ] Link 24 - code:    400
  url:     https://uptane.github.io/docs/1.0.0/standard/uptane-standard##tuf-spec
  parent:  https://uptane.github.io/docs/1.0.0/standard/uptane-standard
  error:   Bad request

- [ ] Link 25 - code:    0
  url:     https://uptane.github.io/blog/blog Index 0
  parent:  https://uptane.github.io/blog
  error:   URL can't contain control characters. '/blog/blog Index 0' (found at least ' ')

**Comment by Abhijay007** [comment]
Here is list of broken links : 

- code:    404
  url:     https://www.d2pmagazine.com/2020/04/02/6099/
  parent:  https://uptane.github.io/docs/learn-more/press
  error:   Not Found

- code:    404
  url:     https://WZ6MBDOHOT-dsn.algolia.net
  parent:  https://uptane.github.io
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/participate.html
  parent:  https://uptane.github.io/docs/learn-more/audits
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/pouf.html
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/reference_pouf.html
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- code:    404
  url:     https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/deployment/introduction.html
  parent:  https://uptane.github.io/docs/1.0.0/standard/uptane-standard
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/attacks.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/overview.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/design.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/governance.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/timeline.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/adoptions.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/people.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/standardref.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/publications.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/audio.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/audits.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/alliance.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    404
  url:     https://uptane.github.io/press.html
  parent:  https://uptane.github.io/deployment-considerations/index.html
  error:   Not Found

- code:    403
  url:     https://rin.org.uk/blogpost/[17](https://github.com/uptane/uptane.github.io/actions/runs/6714594355/job/18248060571#step:5:18)06945/332376/What-is-spoofing-and-how-to-ensure-GPS-security
  parent:  https://uptane.github.io/docs/deployment/best-practices
  error:   Forbidden

- code:    403
  url:     https://knowhow.napaonline.com/how-many-parts-are-in-a-car/
  parent:  https://uptane.github.io//docs/2.0.0/deployment/best-practices#10-changelog
  error:   Forbidden

- code:    400
  url:     https://www.googletagmanager.com
  parent:  https://uptane.github.io
  error:   Bad Request

- code:    400
  url:     https://uptane.github.io/docs/1.0.0/standard/uptane-standard##tuf-spec
  parent:  https://uptane.github.io/docs/1.0.0/standard/uptane-standard
  error:   Bad request

- code:    0
  url:     https://uptane.github.io/blog/blog Index 0
  parent:  https://uptane.github.io//blog
  error:   URL can't contain control characters. '/blog/blog Index 0' (found at least ' ')

**Comment by Mikerniker** [comment]
Hi @Abhijay007 / @hexsecs , if this is still open, can I try this? Also, can you help clarify the following, please see notes:


1.
Parent: https://uptane.github.io/docs/learn-more/press
Link Location: Open Source Framework Helps Automakers Secure OTA Updates-April 2, 2020
Url: https://www.d2pmagazine.com/2020/04/02/6099/
**Notes**: A SUBSCRIPTION is needed TO ACCESS the ARTICLE
**Question**: Since it is behind a paywall, what to do?


2.
Parent: https://uptane.github.io/docs/learn-more/audits 
Location: First paragraph (phrase: Please reach out to us) 
To Fix: <a href="https://uptane.github.io/participate.html" target="_blank" rel="noopener noreferrer">reach out to us</a> 
**Question**: What link should be put here to contact Uptane?

## BROKEN LINKS 3-4:  Issues with the HEAD section of HTML

3. 
Parent: https://uptane.github.io
url: https://WZ6MBDOHOT-dsn.algolia.net
**Notes**: Error is within HEAD section of HTML
To fix: <link data-rh="true" rel="preconnect" href="https://WZ6MBDOHOT-dsn.algolia.net" crossorigin="anonymous">
**Question**: Since this connects to algolia, what should be here instead or should it be deleted? 

4
Parent: https://uptane.github.io
url: https://www.googletagmanager.com
Location: HEAD of HTML 
To Fix: <link rel="preconnect" href="https://www.googletagmanager.com">
**Question**: Since this is located in the HEAD of the html, is this supposed to be deleted or replaced with something else? 


## BROKEN LINKS 5-7: Questions about Best Practices Page and Link to POUFS

**Question**: Should the POUF links be changed to: 
a. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf-main.md
b. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf1.md
c. other: _______

5. 
Parent: https://uptane.github.io/docs/deployment/best-practices
url: https://uptane.github.io/pouf.html
Location Paragraph 3.4 Specifying Wireline Formats
Phrase to fix: POUF Purpose and Guidelines

6. 
Parent: https://uptane.github.io/docs/deployment/best-practices
Location: Paragraph 3.4 Specifying Wireline Formats
Phrase to fix: Uptane Reference Implementation
HTML to Fix: `<a href="https://uptane.github.io/reference_pouf.html" target="_blank" rel="noopener noreferrer">Uptane Reference Implementation</a>`

7.
Parent: https://uptane.github.io/docs/deployment/best-practices
Location: Paragraph 3.4 Specifying Wireline Formats
Phrase to fix: `ASN.1/DER`
HTML to Fix: `<a href="https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats" target="_blank" rel="noopener noreferrer">ASN.1/DER</a>`

## BROKEN LINKS 8-9 : Questions about ISSUES WITH VERSIONS

8. 
Parent: https://uptane.github.io/docs/1.0.0/standard/uptane-standard
Location: Version 1.0.0
Phrase to fix: ``` [DEPLOY] Members of the Uptane Alliance Community, ., "Uptane Deployment Guidelines", n.d..```
To Fix: `Section 6.2 Informative References` <a href="https://uptane.github.io/deployment/introduction.html" target="_blank" rel="noopener noreferrer">Uptane Deployment Guidelines</a>
**Notes**: 
a. The broken link is in version 1.0.0 what should the correct link be?  
b. The link in the current version 2.1.0 is working correctly, and redirects to [this link](https://uptane.github.io/deployment-considerations/index.html) -- is this the link to put in version 1.0.0 ?


9
Parent: https://uptane.github.io//docs/2.0.0/deployment/best-practices#10-changelog
url: https://knowhow.napaonline.com/how-many-parts-are-in-a-car/
**Notes**:
- PARENT NOT WORKING
- URL IS WORKING
**Question**: This seems to be an issue with the 2.0.0 version, currently the url is available within the `2.0.0 version`, however, the `2.1.0` does not appear to have this url.
More specifically:
In `Version 2.0.0`, in `Section 8.0.2 (Integrating supply chain security into Uptain)`. The 2nd paragraph: 

```Developing a secure software supply chain for the auto industry is complicated by a number of factors. For starters, an automobile has, on average, some [30,000 individual components],which can be sourced from perhaps hundreds of individual suppliers.```

has been deleted in version 2.1.0, which contains the link (in version 2.0.0 the link works fine, but the link no longer exists in version 2.1.0). So what kind of fix is preferred for this issue: 
a. ignore 
b. add missing paragraph to version 2.1.0 
c. ______?


## BROKEN LINKS 10-22 : Questions about NAVBAR ISSUES

- For BROKEN LINKS 10-22 (see below) multiple broken links are within the NAVBAR of https://uptane.github.io/deployment-considerations/index.html, I can't seem to locate where to modify the NAVBAR of this particular page.
**Some notes :**
a. The design of the [parent site](https://uptane.github.io/deployment-considerations/index.html) looks very different from the current site (is this an older version or is it in a different repo?)
b. According to @hexsecs comment in [#Issue 104](https://github.com/uptane/uptane.github.io/issues/104) the parent site ([deployment-considerations](https://uptane.github.io/deployment-considerations/index.html) was renamed to [best practices](
https://github.com/uptane/uptane.github.io/blob/main/docs/deployment/best-practices.md). However, the best-practices link in the [NEW docusaurus site](https://uptane.github.io/docs/deployment/best-practices) does not appear to have the same NAVBAR issues where most of the broken links are in [the other site](https://uptane.github.io/deployment-considerations/index.html).
c. Where to locate NAVBAR for this page?
d. Should the broken links just be replaced by the corresponding links in the [new website](https://uptane.github.io/)?

10.
Parent: https://uptane.github.io/deployment-considerations/index.html
URL:  https://uptane.github.io/attacks.html  
Location: NAVBAR The Basics > Why Uptane is Needed

11. 
Parent: https://uptane.github.io/deployment-considerations/index.html
URL: https://uptane.github.io/overview.html   
Location: NAVBAR: The Basics > Project Overview

12.
Parent: https://uptane.github.io/deployment-considerations/index.html
URL: https://uptane.github.io/design.html    
Location: NAVBAR: The Basics > Uptane Design

13.
Parent: https://uptane.github.io/deployment-considerations/index.html
url: https://uptane.github.io/governance.html
Location: NAVBAR: The Basics > Governance

14.
Parent: https://uptane.github.io/deployment-considerations/index.html
url:  https://uptane.github.io/timeline.html
Location: NAVBAR: The Basics > Timeline

15
Parent: https://uptane.github.io/deployment-considerations/index.html
url:   https://uptane.github.io/adoptions.html
Location: NAVBAR: Learn More > Adoptions

16
Parent: https://uptane.github.io/deployment-considerations/index.html
url:   https://uptane.github.io/people.html
Location: NAVBAR: The Basics > People

17
Parent: https://uptane.github.io/deployment-considerations/index.html
url:   https://uptane.github.io/standardref.html
Location: NAVBAR:Design Documents > Uptane Standards Reference Center

18
Parent: https://uptane.github.io/deployment-considerations/index.html
url:   https://uptane.github.io/publications.html
Location: NAVBAR: Learn More > Conference and Journal Publications

19
Parent: https://uptane.github.io/deployment-considerations/index.html
url: https://uptane.github.io/audio.html
Location: NAVBAR: Learn More > Audio Visuals

20
Parent: https://uptane.github.io/deployment-considerations/index.html
url: https://uptane.github.io/audits.html
Location: NAVBAR: Learn More > Security Audits

21
Parent: https://uptane.github.io/deployment-considerations/index.html
url: https://uptane.github.io/alliance.html
Location: NAVBAR: The Basics > Uptane: A JDF/Linux Foundation Project

22
Parent: https://uptane.github.io/deployment-considerations/index.html
url: https://uptane.github.io/press.html
Location: Navbar: PRESS
`<a href="/press.html">Press</a>`

## BROKEN LINKS - : Other Broken Link Issues and Comments

23
Parent: https://uptane.github.io/docs/deployment/best-practices
url: https://rin.org.uk/blogpost/[17](https://github.com/uptane/uptane.github.io/actions/runs/6714594355/job/18248060571#step:5:18)06945/332376/What-is-spoofing-and-how-to-ensure-GPS-security
Phrase to fix: ```here```  (2nd `here`, para 3, 3.1.1 External sources of time)
**Notes**: The link currently seems to work, not sure what to fix.


24
Parent: https://uptane.github.io/docs/1.0.0/standard/uptane-standard
url: https://uptane.github.io/docs/1.0.0/standard/uptane-standard##tuf-spec
Location Phrase: 5.4.4.2 Full verification, paragraph 3: [TUF-spec]
To Fix: <a href="##tuf-spec">[TUF-spec]</a> 
**Question**: From what I understand, I think TUF-spec links to a section within the page itself, should it just be one hashtag or is it supposed to link to something else?


25
Parent: https://uptane.github.io//blog
url: https://uptane.github.io/blog/blog Index 0
**ISSUE**: PARENT NOT WORKING
**Notes**: 
a. I think the issue with the parent here is that it has a double slash: `//blog`
The [current docusaurus site](https://uptane.github.io/blog) uses a single slash: `/blog`  --> which works fine.
b. Also, it seems the words "Index 0" was added after the url -- not sure where to locate this within the repo.
c. I think I may need more clarification on how or where to fix this since the issue seems to be with the parent.

**Comment by Abhijay007** [comment]
sure @Mikerniker go ahead, @hexsecs can you please verify the broken links doubts related to external links

**Comment by Abhijay007** [comment]
> Hi @Abhijay007 / @hexsecs , if this is still open, can I try this? Also, can you help clarify the following, please see notes:
> 
> 
> Parent: https://uptane.github.io/docs/learn-more/press Link Location: Open Source Framework Helps Automakers Secure OTA Updates-April 2, 2020 Url: https://www.d2pmagazine.com/2020/04/02/6099/ **Notes**: A SUBSCRIPTION is needed TO ACCESS the ARTICLE **Question**: Since it is behind a paywall, what to do?
> 
> 
> Parent: https://uptane.github.io/docs/learn-more/audits Location: First paragraph (phrase: Please reach out to us) To Fix: [reach out to us](https://uptane.github.io/participate.html) **Question**: What link should be put here to contact Uptane?
> 
> ## BROKEN LINKS 3-4: Issues with the HEAD section of HTML
> 
> Parent: https://uptane.github.io url: https://WZ6MBDOHOT-dsn.algolia.net **Notes**: Error is within HEAD section of HTML To fix: **Question**: Since this connects to algolia, what should be here instead or should it be deleted?
> 
> 4 Parent: https://uptane.github.io url: https://www.googletagmanager.com Location: HEAD of HTML To Fix: **Question**: Since this is located in the HEAD of the html, is this supposed to be deleted or replaced with something else?
> 
> ## BROKEN LINKS 5-7: Questions about Best Practices Page and Link to POUFS
> **Question**: Should the POUF links be changed to: a. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf-main.md b. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf1.md c. other: _______
> 
> 
> Parent: https://uptane.github.io/docs/deployment/best-practices url: https://uptane.github.io/pouf.html Location Paragraph 3.4 Specifying Wireline Formats Phrase to fix: POUF Purpose and Guidelines
> 
> 
> Parent: https://uptane.github.io/docs/deployment/best-practices Location: Paragraph 3.4 Specifying Wireline Formats Phrase to fix: Uptane Reference Implementation HTML to Fix: `<a href="https://uptane.github.io/reference_pouf.html" target="_blank" rel="noopener noreferrer">Uptane Reference Implementation</a>`
> 
> 
> Parent: https://uptane.github.io/docs/deployment/best-practices Location: Paragraph 3.4 Specifying Wireline Formats Phrase to fix: `ASN.1/DER` HTML to Fix: `<a href="https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats" target="_blank" rel="noopener noreferrer">ASN.1/DER</a>`
> 
> ## BROKEN LINKS 8-9 : Questions about ISSUES WITH VERSIONS
> 
> Parent: https://uptane.github.io/docs/1.0.0/standard/uptane-standard Location: Version 1.0.0 Phrase to fix: ` [DEPLOY] Members of the Uptane Alliance Community, ., "Uptane Deployment Guidelines", n.d..` To Fix: `Section 6.2 Informative References` [Uptane Deployment Guidelines](https://uptane.github.io/deployment/introduction.html) **Notes**: a. The broken link is in version 1.0.0 what should the correct link be? b. The link in the current version 2.1.0 is working correctly, and redirects to [this link](https://uptane.github.io/deployment-considerations/index.html) -- is this the link to put in version 1.0.0 ?
> 
> 9 Parent: https://uptane.github.io//docs/2.0.0/deployment/best-practices#10-changelog url: https://knowhow.napaonline.com/how-many-parts-are-in-a-car/ **Notes**:
> 
> * PARENT NOT WORKING
> * URL IS WORKING
>   **Question**: This seems to be an issue with the 2.0.0 version, currently the url is available within the `2.0.0 version`, however, the `2.1.0` does not appear to have this url.
>   More specifically:
>   In `Version 2.0.0`, in `Section 8.0.2 (Integrating supply chain security into Uptain)`. The 2nd paragraph:
> 
> `Developing a secure software supply chain for the auto industry is complicated by a number of factors. For starters, an automobile has, on average, some [30,000 individual components],which can be sourced from perhaps hundreds of individual suppliers.`
> 
> has been deleted in version 2.1.0, which contains the link (in version 2.0.0 the link works fine, but the link no longer exists in version 2.1.0). So what kind of fix is preferred for this issue: a. ignore b. add missing paragraph to version 2.1.0 c. ______?
> 
> ## BROKEN LINKS 10-22 : Questions about NAVBAR ISSUES
> * For BROKEN LINKS 10-22 (see below) multiple broken links are within the NAVBAR of https://uptane.github.io/deployment-considerations/index.html, I can't seem to locate where to modify the NAVBAR of this particular page.
>   **Some notes :**
>   a. The design of the [parent site](https://uptane.github.io/deployment-considerations/index.html) looks very different from the current site (is this an older version or is it in a different repo?)
>   b. According to @hexsecs comment in [#Issue 104](https://github.com/uptane/uptane.github.io/issues/104) the parent site ([deployment-considerations](https://uptane.github.io/deployment-considerations/index.html) was renamed to [best practices](https://github.com/uptane/uptane.github.io/blob/main/docs/deployment/best-practices.md). However, the best-practices link in the [NEW docusaurus site](https://uptane.github.io/docs/deployment/best-practices) does not appear to have the same NAVBAR issues where most of the broken links are in [the other site](https://uptane.github.io/deployment-considerations/index.html).
>   c. Where to locate NAVBAR for this page?
>   d. Should the broken links just be replaced by the corresponding links in the [new website](https://uptane.github.io/)?
> 
> 
> Parent: https://uptane.github.io/deployment-considerations/index.html URL: https://uptane.github.io/attacks.html Location: NAVBAR The Basics > Why Uptane is Needed
> 
> 
> Parent: https://uptane.github.io/deployment-considerations/index.html URL: https://uptane.github.io/overview.html Location: NAVBAR: The Basics > Project Overview
> 
> 
> Parent: https://uptane.github.io/deployment-considerations/index.html URL: https://uptane.github.io/design.html Location: NAVBAR: The Basics > Uptane Design
> 
> 
> Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/governance.html Location: NAVBAR: The Basics > Governance
> 
> 
> Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/timeline.html Location: NAVBAR: The Basics > Timeline
> 
> 15 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/adoptions.html Location: NAVBAR: Learn More > Adoptions
> 
> 16 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/people.html Location: NAVBAR: The Basics > People
> 
> 17 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/standardref.html Location: NAVBAR:Design Documents > Uptane Standards Reference Center
> 
> 18 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/publications.html Location: NAVBAR: Learn More > Conference and Journal Publications
> 
> 19 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/audio.html Location: NAVBAR: Learn More > Audio Visuals
> 
> 20 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/audits.html Location: NAVBAR: Learn More > Security Audits
> 
> 21 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/alliance.html Location: NAVBAR: The Basics > Uptane: A JDF/Linux Foundation Project
> 
> 22 Parent: https://uptane.github.io/deployment-considerations/index.html url: https://uptane.github.io/press.html Location: Navbar: PRESS `<a href="/press.html">Press</a>`
> 
> ## BROKEN LINKS - : Other Broken Link Issues and Comments
> 23 Parent: https://uptane.github.io/docs/deployment/best-practices url: [https://rin.org.uk/blogpost/[17](https://github.com/uptane/uptane.github.io/actions/runs/6714594355/job/18248060571#step:5:18)06945/332376/What-is-spoofing-and-how-to-ensure-GPS-security](https://rin.org.uk/blogpost/%5B17%5D(https://github.com/uptane/uptane.github.io/actions/runs/6714594355/job/18248060571#step:5:18)06945/332376/What-is-spoofing-and-how-to-ensure-GPS-security) Phrase to fix: `here` (2nd `here`, para 3, 3.1.1 External sources of time) **Notes**: The link currently seems to work, not sure what to fix.
> 
> 24 Parent: https://uptane.github.io/docs/1.0.0/standard/uptane-standard url: https://uptane.github.io/docs/1.0.0/standard/uptane-standard##tuf-spec Location Phrase: 5.4.4.2 Full verification, paragraph 3: [TUF-spec] To Fix: [[TUF-spec]](##tuf-spec) **Question**: From what I understand, I think TUF-spec links to a section within the page itself, should it just be one hashtag or is it supposed to link to something else?
> 
> 25 Parent: https://uptane.github.io//blog url: https://uptane.github.io/blog/blog Index 0 **ISSUE**: PARENT NOT WORKING **Notes**: a. I think the issue with the parent here is that it has a double slash: `//blog` The [current docusaurus site](https://uptane.github.io/blog) uses a single slash: `/blog` --> which works fine. b. Also, it seems the words "Index 0" was added after the url -- not sure where to locate this within the repo. c. I think I may need more clarification on how or where to fix this since the issue seems to be with the parent.

@Mikerniker most of the links are with in the docs, we just need to point those links to the new location rather than fixing the text of previously broken links

**Comment by Abhijay007** [comment]
for example 

the correct link for this should be : 

code: 404
url: https://uptane.github.io/participate.html
parent: https://uptane.github.io/docs/learn-more/audits
error: Not Found

 - > https://uptane.github.io/docs/learn-more/participate

**Comment by hexsecs** [comment]
@Mikerniker  -

Parent: https://uptane.github.io/docs/learn-more/press
Link Location: Open Source Framework Helps Automakers Secure OTA Updates-April 2, 2020
Url: https://www.d2pmagazine.com/2020/04/02/6099/

This site seems down. Let's just reference the archive.org version of the article.
https://web.archive.org/web/20230703104528/https://www.d2pmagazine.com/2020/04/02/6099/

**Comment by hexsecs** [comment]
Ok. Somehow the old deployment considerations site is still in the active tree. I think this is a *major* source of many of these broken links.

This page shouldn't even exist...
https://uptane.github.io/deployment-considerations/index.html

**Comment by hexsecs** [comment]
> BROKEN LINKS 5-7: Questions about Best Practices Page and Link to POUFS
> Question: Should the POUF links be changed to:
> a. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf-main.md
> b. https://github.com/uptane/uptane.github.io/blob/main/docs/enhancements/pouf/pouf1.md
> c. other: _______

https://uptane.github.io/docs/enhancements/pouf/pouf-main <-- use this page (relative link form)

> 
> Parent: https://uptane.github.io/docs/deployment/best-practices
> url: https://uptane.github.io/pouf.html
> Location Paragraph 3.4 Specifying Wireline Formats
> Phrase to fix: POUF Purpose and Guidelines

https://uptane.github.io/docs/enhancements/pouf/pouf-main <-- use this page (relative link form)

> 
> Parent: https://uptane.github.io/docs/deployment/best-practices
> Location: Paragraph 3.4 Specifying Wireline Formats
> Phrase to fix: Uptane Reference Implementation
> HTML to Fix: <a href="https://uptane.github.io/reference_pouf.html" target="_blank" rel="noopener noreferrer">Uptane Reference Implementation</a>

https://uptane.github.io/docs/enhancements/pouf/pouf1 <-- use this page (relative link form)


> 
> Parent: https://uptane.github.io/docs/deployment/best-practices
> Location: Paragraph 3.4 Specifying Wireline Formats
> Phrase to fix: ASN.1/DER
> HTML to Fix: <a href="https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats" target="_blank" rel="noopener noreferrer">ASN.1/DER</a>

https://uptane.github.io/docs/enhancements/pouf/pouf1#file-formats <-- use this page (relative link form)

**Comment by hexsecs** [comment]
> # BROKEN LINKS 8-9 : Questions about ISSUES WITH VERSIONS
> Parent: https://uptane.github.io/docs/1.0.0/standard/uptane-standard Location: Version 1.0.0 Phrase to fix:  [DEPLOY] Members of the Uptane Alliance Community, ., "Uptane Deployment Guidelines", n.d.. To Fix: Section 6.2 Informative References [Uptane Deployment Guidelines](https://uptane.github.io/deployment/introduction.html) Notes: a. The broken link is in version 1.0.0 what should the correct link be? b. The link in the current version 2.1.0 is working correctly, and redirects to [this link](https://uptane.github.io/deployment-considerations/index.html) -- is this the link to put in version 1.0.0 ?

I understand what happened here. When the project began, we did not version deployment considerations (now best practices). Let's just point to the head.
https://uptane.github.io/docs/deployment/best-practices


> 9 Parent: https://uptane.github.io//docs/2.0.0/deployment/best-practices#10-changelog url: https://knowhow.napaonline.com/how-many-parts-are-in-a-car/ Notes:

> PARENT NOT WORKING
> URL IS WORKING
> Question: This seems to be an issue with the 2.0.0 version, currently the url is available within the 2.0.0 version, however, the 2.1.0 does not appear to have this url.
> More specifically:
> In Version 2.0.0, in Section 8.0.2 (Integrating supply chain security into Uptain). The 2nd paragraph:
> Developing a secure software supply chain for the auto industry is complicated by a number of factors. For starters, an automobile has, on average, some [30,000 individual components],which can be sourced from perhaps hundreds of individual suppliers.
> 
> has been deleted in version 2.1.0, which contains the link (in version 2.0.0 the link works fine, but the link no longer exists in version 2.1.0). So what kind of fix is preferred for this issue: a. ignore b. add missing paragraph to version 2.1.0 c. ______?

This seems odd. We need to figure out where in the code is serving up the  double '//' in the path.

**Comment by Mikerniker** [comment]
Hi @hexsecs 

Ok, so just to summarize:
Is this correct, and can you help indicate what link to use for number 2 ? 

**Links to Change:**
- 1 Change to: https://web.archive.org/web/20230703104528/https://www.d2pmagazine.com/2020/04/02/6099/
- 2 Broken Link: ```<a href="https://uptane.github.io/participate.html" target="_blank" rel="noopener noreferrer">reach out to us</a>```
Change to: ?
- 5 Change to: https://uptane.github.io/docs/enhancements/pouf/pouf-main 
- 6 Change to: https://uptane.github.io/docs/enhancements/pouf/pouf1  
- 7 Change to: https://uptane.github.io/docs/enhancements/pouf/pouf1#file-formats
- 8 Change to: https://uptane.github.io/docs/deployment/best-practices
- 24 Change to `#tuf-spec` instead of `##tuf-spec`  ? 


**Ignore List:**
- BROKEN LINKS 3-4: Issues with the HEAD section of HTML
  - Reason: From @hexsecs: It might have to do with how the script is run. Both of those are site loading optimizations, can ignore.
- BROKEN LINK 9 : Questions about ISSUES WITH VERSIONS
  - Reason: In version 2.0.0 the link works fine, but the link no longer exists in version 2.1.0
- BROKEN LINKS 10-22 : Questions about NAVBAR ISSUES
  - Reason: From @hexsecs: 
    - Deployment considerations site is still in the active tree, which might be the source of the broken links.
    - Might be an issue with the deployment CI?
- 23 Reason: Link currently seems to work
- 25 Reason: Small typo, the issue was in what was copied into the comment. There is no double `//` in the path. The link seems to work, but it is being blocked by the CI bot.

**Comment by hexsecs** [comment]
@Mikerniker - I think this looks good. If you can submit those, I can always re-run the script after your pull request and we can create a new issue for any remaining items.
